docs/faq/faq.md

```diff
@@ -40,3 +40,11 @@ There are some things you should consider before doing so:
   conflicts.
 
 TODO: Link to adoption use case once it gets on board.
+
+## Why Cluster API doesn't have the CRD yaml file?
+
+Kubernetes provides two methods to extend APIs: Custom Resource and Kubernetes API Aggregation Layer, for more detail, we can refer to [Extending the Kubernetes API](https://kubernetes.io/docs/concepts/extend-kubernetes/).
+
+Karmada uses the two extension methods. For example, `PropagationPolicy` and `ResourceBinding` use the Custom Resource method, and `Cluster` resource use the `Kubernetes API Aggregation Layer` method.
```

Comment:
It would be great to explain a little bit more details about why Cluster needs to be extended by aggregated layer.

Reply:
Ok, let me do it.